$MirOS: ports/net/wget/patches/patch-src_ftp-ls_c,v 1.2 2005/06/29 14:04:46 tg Exp $
--- src/ftp-ls.c.orig	Thu Aug  4 21:52:33 2005
+++ src/ftp-ls.c	Thu Nov 10 21:23:08 2005
@@ -903,7 +903,7 @@ ftp_index (const char *file, struct url 
 	  };
 	  struct tm *ptm = localtime ((time_t *)&f->tstamp);
 
-	  fprintf (fp, "%d %s %02d ", ptm->tm_year + 1900, months[ptm->tm_mon],
+	  fprintf (fp, "%lld %s %02d ", (long long)ptm->tm_year + 1900, months[ptm->tm_mon],
 		  ptm->tm_mday);
 	  if (ptm->tm_hour)
 	    fprintf (fp, "%02d:%02d  ", ptm->tm_hour, ptm->tm_min);
